### PR TITLE
test: Cypress - Flaky test fix

### PR DESCRIPTION
**Pull request by Vijetha-Kaja**
## Description

**Fixed below flaky tests**

- Command_Click_Navigation_spec.js
- List4_spec.js
- Select_spec.js
- PassingParams_Spec.ts

## Type of change

- Flaky test fix

## How Has This Been Tested?
- Cypress test runs

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/ExplorerTests/Entity_Explorer_Widgets_Copy_Paste_Delete_Undo_Keyboard_Event_spec.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ describe("Test Suite to validate copy/delete/undo functionalites", function () {
       200,
     );
     agHelper.Sleep(1000);
-    cy.get("body").type(`{${modifierKey}}z`, { force: true });
+    cy.get("body").type(`{${modifierKey}}z`);
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/ExplorerTests/Entity_Explorer_Widgets_Copy_Paste_Delete_Undo_Keyboard_Event_spec.js:42]: Revert this fix, I have tried to fix it in my other PR.

File: app/client/cypress/integration/Regression_TestSuite/ServerSideTests/Params/PassingParams_Spec.ts [hunk 2]
```diff
@@ -108,13 +124,14 @@ describe("[Bug] - 10784 - Passing params from JS to SQL query should not break",
...
     _.agHelper.ValidateNetworkExecutionSuccess("@postExecute");
-    _.table.ReadTableRowColumnData(0, 0, "v1", 2000).then((cellData) => {
+    _.table.ReadTableRowColumnData(0, 1, "v2", 2000).then((cellData) => {
       expect(cellData).to.be.equal("7");
     });
   });
 
   it("6. With Optional chaining : {{ (() => { return this?.params.condition })() }}", function () {
     _.deployMode.NavigateBacktoEditor();
+    _.agHelper.Sleep(500);
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ServerSideTests/Params/PassingParams_Spec.ts:134]: `NavigateBacktoEditor()` is already waiting for 2000ms & checking if canvas is visible.
Are those check still not enough? Do we need this wait?

Comment (Vijetha-Kaja) [reply app/client/cypress/integration/Regression_TestSuite/ServerSideTests/Params/PassingParams_Spec.ts:134]: Yes, I 've added wait for one test case in this spec and verified before adding it to every test case with navigateBackToEditor() in this spec

**Pull request merged**